packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -116,6 +117,7 @@ class LongPressEndDetails {
   const LongPressEndDetails({
     this.globalPosition = Offset.zero,
     Offset localPosition,
+    this.velocity = Velocity.zero,
```

Comment:
See the commend regarding velocity above, if that's true then we don't need this new field.